src/App.tsx

```diff
@@ -1,52 +1,23 @@
 import './App.css'
 import { AvatarShadowType, Header } from './components/Header';
 import { Links } from "./components/Links";
-import { LinkTheme } from './components/Links';
-import SampleAvatar from "./assets/png/sample-avatar.png";
+import Config from "./data/config.json";
+import AvatarImage from "./assets/png/avatar.png";
 
 function App() {
   return (
     <div className="App">
       <Header
         avatarOptions={{
-          title: "Haneen Mahdin",
-          description: "An aspiring full-stack developer and a UI/UX Designer",
-          avatarImageSource: SampleAvatar,
-        }}
-
-        stylingOptions={{
-          shadowType: AvatarShadowType.USE_DEFAULT_SHADOW,
-          isShadowAnimated: true
+          title: Config.avatarOptions.title,
+          description: Config.avatarOptions.description,
+          avatarImageSource: AvatarImage
```

Comment:
**NOTE:** only use the image from `./assets/png/avatar.png` as if you are passing the image from the json file, it would result in a 404 resource not found error 😢